src/theme/NavbarItem.tsx

```diff
@@ -1,24 +1,55 @@
 import React from 'react'
 import OriginalNavBarItem from '@theme-original/NavbarItem'
 import { useLocation } from '@docusaurus/router'
+import { TraceEvent } from '@uniswap/analytics'
+import { BrowserEvent, EventName, DocsProtocolVersion } from '@uniswap/analytics-events'
 
-export default function NavbarItem(props: { className: string }) {
+enum ProtocolVersion {
+  V1 = 'V1',
+  V2 = 'V2',
+  V3 = 'V3',
+}
+
+const getSection = (version: string) => {
+  if (version === ProtocolVersion.V2) {
+    return DocsProtocolVersion.V2
+  }
+
+  if (version === ProtocolVersion.V1) {
+    return DocsProtocolVersion.V1
+  }
+
+  return DocsProtocolVersion.V3
```

Comment:
1. Should any value default to V3, the literal value if not V3, or some specific value signifying unknown?
2. It would probably be best to use a `switch` statement here
3. We should also order for cleanliness

```suggestion
  switch (version) {
    case ProtocolVersion.V1:
      return DocsProtocolVersion.V1;
    case ProtocolVersion.V2:
      return DocsProtocolVersion.V2;
    case ProtocolVersion.V3:
      return DocsProtocolVersion.V3;
    default:
      return version; // or 'UNKNOWN'?
  }
```